hack/run.sh

```diff
@@ -33,6 +33,7 @@ function usage() {
 	echo "   teardown-chaos                                          Remove chaosduck"
 	echo "   profiler                                                Run profiling tests"
 	echo "   generate                                                Run code generators"
+	echo "   deploy-control-plane                                    Build and deploy control plane tests"
```

Comment:
```suggestion
	echo "   deploy-control-plane                                    Build and deploy control plane"
```
I would remove the "tests" from the description, as we only deploy the control-plane IIUC.